README.md

```diff
@@ -108,6 +109,28 @@ This setup enables local testing of clients such as orchestration and OpenAI, pr
 >
 > For example, while the SAP Cloud SDK for AI uses the [dotenv](https://www.npmjs.com/package/dotenv) library to load environment variables, NextJS uses a [specific configuration](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables) to load them.
 
+## Error Handling
+
+A common error scenario is `Request failed with status code STATUS_CODE` coming from `AxiosError`. 
+In this case, SAP Cloud SDK for AI uses [`ErrorWithCause`](https://sap.github.io/cloud-sdk/docs/js/features/error-handling) to provide more detailed error information.
+
+The following example shows how to access useful information from the error.
+
+```ts
+try {
+  ... // execute request
+} catch (e) {
+  // Print error messages from different layers
```

Comment:
A little bit more detail, perhaps with an example error here would be helpful, even if we already link to the cloud SDK docs here.

Reply:
Improved